pages/news/_title/index.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
pages/news/_title/index.vue -> pages/news/_title.vue

Reply:
@jasmine-sandhu17  why make title optional? I mean we don't have any `news/index` file  index file is `/` home page

Reply:
`pages/news/_title.vue` will get converted into `pages/news/:title` not `pages/news/:title?`

Reply:
@jasmine-sandhu17 It's already `pages/news/:title`
ps  `_directory` means `required`